### PR TITLE
Add information about Metals to scala-lang website and replace Ensime

### PR DESCRIPTION
**Pull request by tgodzik**
Replace Ensime logo with Metals since Ensime is no longer actively maintained and add all supported editors for Metals.

image

### CODE REVIEW

**Comment by jvican** [comment]
What about moving Eclipse to the last position? I'm not sure how often Eclipse is used but given that not many developers are using it according to the Stack Overflow developer surveys we should most likely demote its visibility.

**Comment by tgodzik** [comment]
> What about moving Eclipse to the last position? I'm not sure how often Eclipse is used but given that not many developers are using it according to the Stack Overflow developer surveys we should most likely demote its visibility.

I am ok with that. Any other opinions?

**Comment by tgodzik** [comment]
Moved Eclipse back:
image

File: _layouts/frontpage.html [hunk 1]
```diff
@@ -154,10 +154,10 @@ <h2><span>IDEs for Scala</span></h2>
     <ul>
       {% for ide in page.scalaIDEs %}
       <li>
-        <a {% if ide.ensime == true %}class="masterTooltip" title="{{site.data.common.texts.ensimeSupportedInIDE}}"{% endif %} href="{{ide.url}}" target="_blank">
-          {% if ide.ensime == true %}
+        <a {% if ide.metals == true %}class="masterTooltip" title="{{site.data.common.texts.metalsSupportedInIDE}}"{% endif %} href="{{ide.url}}" target="_blank">
+          {% if ide.metals == true %}
             <span class="bullet">
-              <img src="/resources/img/frontpage/icon-ensime.png" alt="">
+              <img src="/resources/img/frontpage/scalameta-logo.png" alt="">
```
[the remark this answers is not in the document]
Comment (tgodzik) [reply _layouts/frontpage.html]: It's mostly `png` files there anyway and it looks ok right now.

File: index.md [hunk 2]
```diff
@@ -49,25 +49,32 @@ scalaBackends:
 scalaIDEs:
   - name: IntelliJ IDEA
     icon: /resources/img/frontpage/intelliJ.png
-    ensime: false
+    metals: false
     url: "https://www.jetbrains.com/idea/"
+  - name: VS Code
+    icon: /resources/img/frontpage/vs_code.png
+    metals: true
+    url: "https://code.visualstudio.com/"
```
Comment (olafurpg) [inline index.md]:
```suggestion
    url: "https://scalameta.org/metals/docs/editors/vscode.html"
```

File: index.md [hunk 3]
```diff
@@ -49,25 +49,32 @@ scalaBackends:
...
+  - name: GNU Emacs
+    icon: /resources/img/frontpage/emacs-logo.svg
+    metals: true
+    url: "https://www.gnu.org/software/emacs/"
```
Comment (olafurpg) [inline index.md]:
```suggestion
    url: "https://scalameta.org/metals/docs/editors/emacs.html"
```

File: index.md [hunk 4]
```diff
@@ -49,25 +49,32 @@ scalaBackends:
...
+  - name: Vim
+    icon: /resources/img/frontpage/vim-logo.svg
+    metals: true
+    url: "https://www.vim.org/"
```
Comment (olafurpg) [inline index.md]:
```suggestion
    url: "https://scalameta.org/metals/docs/editors/vim.html"
```

File: index.md [hunk 5]
```diff
@@ -49,25 +49,32 @@ scalaBackends:
...
   - name: Sublime Text
     icon: /resources/img/frontpage/sublime.png
-    ensime: true
+    metals: true
     url: "https://www.sublimetext.com/"
```
Comment (olafurpg) [inline index.md]:
```suggestion
    url: "https://scalameta.org/metals/docs/editors/sublime.html"
```

File: index.md [hunk 6]
```diff
@@ -49,25 +49,32 @@ scalaBackends:
...
   - name: Atom
     icon: /resources/img/frontpage/atom.png
-    ensime: true
+    metals: true
     url: "https://atom.io/"
```
Comment (olafurpg) [inline index.md]:
```suggestion
    url: "https://scalameta.org/metals/docs/editors/atom.html"
```

**Comment by tgodzik** [comment]
> How about we link to the installation docs instead of the editor websites?

I was thinking about it, but we previously pointed to the editor pages themselves - might make sense to change that

**Comment by tgodzik** [comment]
> Thanking Ensime and the Ensime maintainers for the editor support provided over many years, I agree it makes sense to update the website to link to (and promote) Metals.

That should be said by me :man_facepalming:  Very sorry for being undiplomatic. It goes without saying that Ensime was an awesome project and proved that tooling can be cool :rocket:

**Pull request merged**